### ISSUE TITLE
Console not available under a function

### ISSUE THREAD

**Issue by askpatrickw** [Azure/iotc-device-bridge#48]
When I open the function created, there is no Console tab. (step 3)
There IS A console tab under the AppService.
From there I cd into IOTCIntegraton and ran `npm install`

Everything seems to be working... so I think the README just needs a refresh.

**Comment by peolivei2** [comment]
Thanks for reporting it. Looks like the UI has changed significantly. I'll update the README.

**Comment by topese** [comment]
I have deployed the Device Bridge a number of times the previously stated way, with the Console under the actual Function. When I tried again today I took some time to actually realize the Console had been moved.

When I tried to deploy the function as stated in the new updated README, everything seems to deploy just fine. When I do the step >> npm install I get a some WARN that are unseen by me previously. 

![Screen Shot 2020-06-17 at 03 00 04](https://user-images.githubusercontent.com/29483057/84842904-b7702200-b046-11ea-8c47-25c930733973.png)

When I try to post some JSON data into the URL I get a 500 Internal Server Error back, for all messages. 

Is there possibly an issue with the used deploy script or something that is not working as before?
